util/libeay.num

```diff
@@ -4370,7 +4370,7 @@ DH_compute_key_padded                   4732	EXIST::FUNCTION:DH
 ECDSA_METHOD_set_sign                   4733	EXIST::FUNCTION:ECDSA
 CMS_RecipientEncryptedKey_cert_cmp      4734	EXIST:!VMS:FUNCTION:CMS
 CMS_RecipEncryptedKey_cert_cmp          4734	EXIST:VMS:FUNCTION:CMS
-DH_KDF_X9_42                            4735	EXIST::FUNCTION:DH
+DH_KDF_X9_42                            4735	EXIST::FUNCTION:CMS,DH
```

Comment:
I *do* understand the implications. But consider case when you have application (that pays attention to returned error codes) linked with shared library and with function in question. Do we *have* to punish it if library provider has taken away DH_KDF_X9_42? Of course one can argue that nobody does that, i.e. have separate application binary and OpenSSL shared library providers, but then question is why aren't there any? Is it possible that because of this lax attitude that nobody does it? Wouldn't it actually benefit users?

[Yes, this kind of calls for question if *any* of #if NO_THIS_OR_THAT are justified in public headers or shared library export table...]